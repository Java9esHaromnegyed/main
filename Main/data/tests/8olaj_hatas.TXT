loadArena testArena8.txt
initArena
tick 1
robotMovement 1 UP
tick 1