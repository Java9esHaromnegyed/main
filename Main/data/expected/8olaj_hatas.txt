arenaLoaded src: testArena8.txt
robotAdded id: 1 name: „player1” pos: [0; 0]
obstacleAdded type: OIL pos: [1; 0]
arenaInited
tick
robotMoved id: 1 pos: [1; 0]
robotParalized id: 1
tick
robotMoved id: 1 pos: [2; 0]